server/routes/healthy.js

```diff
@@ -9,10 +9,8 @@ module.exports = {
   options: {
     handler: async (request, h) => {
       try {
-        if (await databaseService.isConnected()) {
-          return h.response('ok').code(OK)
-        }
-        return h.response('database unavailable').code(SERVICE_UNAVAILABLE)
+        await (await sequelize).authenticate()
+        return h.response('ok').code(OK)
       } catch (ex) {
         console.error('error running healthy check', ex)
         return h.response(`error running healthy check: ${ex.message}`).code(SERVICE_UNAVAILABLE)
```

Comment:
You're potentially losing a bit of info about the reason the heath check failed but I assume (and I think it's a sensible assumption) that the exception message contains some info that mentions the database connection failed. So I'm fine with this if you agree with that assumption.

Reply:
Previously, if `authenticate` threw an error it would have logged the message in `database-service` and returned false which then would have returned the `database unavailable` message to the web request. Now, if `authenticate` throws an error it will log the error and return the error message to the web request. In which case, a bit more information is returned to the web request.